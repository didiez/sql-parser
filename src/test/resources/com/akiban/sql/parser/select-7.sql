SELECT * FROM t1,t2 FULL OUTER JOIN t3 ON t2.x = t3.x FULL JOIN t4 ON t3.x = t4.x