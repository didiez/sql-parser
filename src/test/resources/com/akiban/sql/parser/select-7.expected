com.akiban.sql.parser.CursorNode@6564dbd5
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	com.akiban.sql.parser.SelectNode@409cb0f4
	isDistinct: false
	resultColumns: 		
		com.akiban.sql.parser.ResultColumnList@6f25844f

		[0]:		
		com.akiban.sql.parser.AllResultColumn@811978b
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		com.akiban.sql.parser.FromList@3af42ad0

		[0]:		
		com.akiban.sql.parser.FromBaseTable@7426dbec
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
		[1]:		
		com.akiban.sql.parser.FullOuterJoinNode@23309e87
		fullOuterJoin: true
		nestedInParens: false
		correlation Name: null
		null
		leftResultSet: 			
			com.akiban.sql.parser.FullOuterJoinNode@4cf7c31d
			fullOuterJoin: true
			nestedInParens: false
			correlation Name: null
			null
			leftResultSet: 				
				com.akiban.sql.parser.FromBaseTable@1e2acc65
				tableName: t2
				updateOrDelete: null
				null
				correlation Name: null
				null
			rightResultSet: 				
				com.akiban.sql.parser.FromBaseTable@bd10a5c
				tableName: t3
				updateOrDelete: null
				null
				correlation Name: null
				null
			joinClause: 				
				com.akiban.sql.parser.BinaryRelationalOperatorNode@48ff2413
				operator: =
				methodName: equals
				type: null
				leftOperand: 					
					com.akiban.sql.parser.ColumnReference@669980d5
					columnName: x
					tableName: t2
					type: null
				rightOperand: 					
					com.akiban.sql.parser.ColumnReference@67d95492
					columnName: x
					tableName: t3
					type: null
		rightResultSet: 			
			com.akiban.sql.parser.FromBaseTable@6d3f7281
			tableName: t4
			updateOrDelete: null
			null
			correlation Name: null
			null
		joinClause: 			
			com.akiban.sql.parser.BinaryRelationalOperatorNode@6076ab2f
			operator: =
			methodName: equals
			type: null
			leftOperand: 				
				com.akiban.sql.parser.ColumnReference@ee0cc23
				columnName: x
				tableName: t3
				type: null
			rightOperand: 				
				com.akiban.sql.parser.ColumnReference@487a1576
				columnName: x
				tableName: t4
				type: null
